### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
# Pull Request

## What does this PR do?

Found via `codespell -L crate,nam,hart`.

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: meilisearch-lib/src/dump/compat/v3.rs [hunk 1]
```diff
@@ -134,7 +134,7 @@ impl Processed {
     fn update_task(self, task: &mut Task) {
         self.from.update_task(task);
 
-        let event = TaskEvent::Succeded {
```
Comment (Kerollmops) [inline meilisearch-lib/src/dump/compat/v3.rs:137]: This is an enum variant that we want to keep that way as it must be compatible with the previous versions that have this typo.

File: meilisearch-lib/src/dump/compat/v4.rs [hunk 2]
```diff
@@ -38,7 +38,7 @@ pub enum TaskEvent {
         batch_id: BatchId,
     },
     Processing(#[serde(with = "time::serde::rfc3339")] OffsetDateTime),
-    Succeded {
```
Comment (Kerollmops) [inline meilisearch-lib/src/dump/compat/v4.rs:41]: Same here. Could you remove this typo, please?

File: meilisearch-lib/src/dump/compat/v4.rs [hunk 3]
```diff
@@ -62,7 +62,7 @@ impl From<TaskEvent> for NewTaskEvent {
                 batch_id,
             },
             TaskEvent::Processing(x) => NewTaskEvent::Processing(x),
-            TaskEvent::Succeded { result, timestamp } => {
```
Comment (Kerollmops) [inline meilisearch-lib/src/dump/compat/v4.rs:65]: And here. Could you keep this typo?

**Review by Kerollmops — CHANGES_REQUESTED** [review]

**Comment by curquiza** [comment]
Hello @kianmeng 

Do you plan to participate in Hacktoberfest? We can wait for 1st October if you want to merge the PR so that you will have at least 1 contribution

**Comment by kianmeng** [comment]
@curquiza Noted, but I'm not interested in Hacktoberfest, you can merge it now if you want. Thanks for the offering though. :smile:

**Comment by curquiza** [comment]
Noted! @Kerollmops I let you merge once you ok with the PR!

**Review by Kerollmops — APPROVED** [review]
Thank you!
bors merge

**Comment by curquiza** [comment]
Thank you for contributing to Meilisearch. If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some swag from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**